### PR TITLE
Update show dont tell

### PR DESCRIPTION
**Pull request by anttiviljami**
Changes:

- Focus on delivering working software early and often. Previous version was too focused on Prototypes and Proof-of-Concepts.
- Add our expectation of maintaining shippable code. Avoid creating tech debt that we plan to fix later.
- Reminder that as engineers we should not let our users remind us of the basics. We know what quality software looks like.

Screenshot 2021-09-28 at 11 27 49
.

### CODE REVIEW

**Review by jpinho — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -50,6 +57,20 @@ We expect teams to automate testing, deployment and monitoring of their features
...
+
+While prototypes and proof-of-concepts can be great tools early on, we should always maintain shippable code. Avoid creating technical debt. Don't depend on polish happening later. 
+
+Gather feedback where uncertainty exists. We are our own best testing team and as software professionals we shouldn't let our users remind us what quality software looks like. Don't ship and apologise for incomplete or broken features. Ship [minimum lovable products](https://productschool.com/blog/product-management-2/minimum-lovable-product/) instead.
+
+### Why?
+
+- **Code wins arguments**: Debating abstract ideas leads to endless meetings. A working piece of software is always better than architecture diagrams and RFCs.
+- **Move fast**: Putting feedback loops in place early on helps us make more informed decisions collaboratively and commit to deliverables that create tangible progress.
+- **Risk management**: Even with thorough design work and great planning, unplanned work is guaranteed to appear once you're in the process of building. So better start early.
```
Comment (jpinho) [inline README.md]:
```suggestion
- **Risk management**: Even with throughout design work and great planning, unplanned work is guaranteed to appear once you're in the process of building. So better start early. 
```

Comment (jpinho) [reply README.md]: Little typo.

**Review by jpinho — APPROVED** [review]
LGTM! MAX THE US!

File: README.md (hunk 1 above)
Comment (anttiviljami) [reply README.md]: Actually not a typo.

I meant [thorough](https://dictionary.cambridge.org/dictionary/english/thorough) as in careful. 

But maybe for simpler language careful would be better 🙌

**Pull request merged**